docs/public-networks/tutorials/besu-teku-mainnet.md

```diff
@@ -0,0 +1,205 @@
+---
+Description: How to run Besu and Teku on Mainnet
+---
+
+# Run Besu and Teku on Mainnet
+
+Run Besu as an [execution client](../concepts/the-merge.md#execution-clients) and
+[Teku](https://docs.teku.consensys.net/en/stable/)
+as a [consensus client](../concepts/the-merge.md#consensus-clients) on Ethereum Mainnet.
+
+## 1. Install Besu and Teku
+
+Install [Besu](../get-started/install/binary-distribution.md) and
+[Teku](https://docs.teku.consensys.net/en/stable/HowTo/Get-Started/Installation-Options/Install-Binaries/).
+
+Ensure you meet the prerequisites for the installation option you use.
+For example, you must have Java 11+ if using the Besu and Teku binary distributions.
+
+Ensure you meet the [system requirements for Besu on public networks](../get-started/system-requirements.md).
+
+## 2. Generate the shared secret
+
+Run the following command:
+
+```bash
+openssl rand -hex 32 | tr -d "\n" > jwtsecret.hex
+```
+
+You will specify `jwtsecret.hex` when starting both Besu and Teku.
+This is a shared JWT secret the clients use to authenticate each other when using the
+[Engine API](../how-to/use-engine-api.md).
+
+## 3. Start Besu
+
+Run the following command or specify the options in a [configuration file](../how-to/configuration-file.md):
+
+```bash
+besu \
+  --sync-mode=X_SNAP           \
+  --data-storage-format=BONSAI \
+  --rpc-http-enabled=true      \
+  --rpc-http-host="0.0.0.0"    \
+  --rpc-ws-enabled=true        \
+  --rpc-ws-host="0.0.0.0"      \
+  --host-allowlist=<IP of Besu node>,127.0.0.1,localhost        \
+  --engine-host-allowlist=<IP of Besu node>,127.0.0.1,localhost \
+  --Xmerge-support=true        \
+  --engine-jwt-secret=<path to jwtsecret.hex>
+```
+
+Specify:
+
+- The path to the `jwtsecret.hex` file generated in [step 2](#2-generate-the-shared-secret) using
+  the [`--engine-jwt-secret`](../reference/cli/options.md#engine-jwt-secret) option.
+- The IP address of your Besu node using the [`--host-allowlist`](../reference/cli/options.md#host-allowlist)
+  and [`--engine-host-allowlist`](../reference/cli/options.md#engine-host-allowlist) options.
+
+This command also specifies the following options:
+
+- [`sync-mode`](../reference/cli/options.md#sync-mode) - Use [snap sync](../get-started/connect/sync-node.md#snap-synchronization).
```

Comment:
missing the `--`.

Can also say something like:

In the command line:

-  `--sync-mode` specifies using snap sync to sync the Besu node.
- ...